SecurityPkg/Library/Tpm2DebugLib/Tpm2DebugLibVerbose.c

```diff
@@ -9,10 +9,14 @@ MU_CHANGE
 
 **/
 
+#include <Uefi.h>
 #include <Library/DebugLib.h>
 #include <Library/BaseLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/Tpm2DebugLib.h>
 
 #include <IndustryStandard/Tpm20.h>
+#include <IndustryStandard/UefiTcgPlatform.h>

```

Comment:
we may need to revert a recent PR which we have decided is not needed which moved more things to this header file.  Might want to check with raymond

Reply:
Will do

Reply:
Raymond's change would have affected this PR, but it went into 202405 and has not been cherry-picked to 202502 yet (and sounds like it won't be). So the headers here are correct.